### DISCUSSION TITLE
[Questions] How do some spells work?

### DISCUSSION THREAD

**Discussion by LilacRaven** [Sollace/Unicopia#595] [Q&A]
For example, spells like the Mind Swap Gem Dark Vortex do not seem to do anything, unless there's something that needs to be done to make the spells work? 

Additionally, how does the Group Teleporting spell work? Is there a way to activate it, or are is bangles of comradery needed to make it work?

Furthermore, is there a way to acquire the telekinesis spell? 

Oh, and sorry for not labelling this post as a question, I don't know how to, lol.

**Comment by Sollace** [comment]
All the spells have a page description given to them by Starswirl in the spellbook.

I haven't really documented them on the wiki (that's my bad - not had the time to work on it)

Some of the spells require you to use them with the shoot rather than cast ability, and some operate on what you're looking at. For example the mind swap will swap with the entity you're looking at (up to 100 blocks away) when you use the cast ability to cast it, or will swap with the entity that you hit when using the shoot ability.

Dark vortex creates a black hole. It's a placed spell so you have to use the cast ability with it, and the hole starts off small and grows as you feed it items and entities, and will generate mana for you.

Group teleport is an ability, not a spell. It's the same as the normal teleport ability but will take anyone nearby that's wearing a bangle of comradery that you have signed.

For telekinesis there isn't a dedicated ability but you can use the attraction spell and give it the focus trait (I think) to be able to cast it and only affect the entity you're looking at to have it follow you.

**Comment by Sollace** [comment]
Also I went and added a template option for questions. ^-^

**Comment by LilacRaven** [comment]
Thank you for answering! 

Both the Dark Vortex Spell and the mindswap spell seem to not be working for me? The text is red for both of them, and when I try to shoot the mind swap spell for example, no blast comes out, all I hear is a magic noise. 

For Dark Vortex, it seems to make a lightning effect every now and again, but no player, mob, or any other entity is effected by it. Is there something I need to do to make it so that these spells work?

**Comment by Sollace** [comment]
You have to feed the dark vortex before it will get large enough to affect players or mobs. If you get close enough and throw items directly into it, it should begin to grow.

The mind swap spell definitely works 1.21 as it was one of the spells I tested when updating. There is a bug in the current release that will cause some spells to not save/cancel which I've fixed but not yet released. Going to release that once I've checked some of the other issues that have been reported.

**Comment by LilacRaven** [comment]
Ah, I see. I mentioned the Mind Swap Gem as when I try to use it by firing it, it doesn't seem to do anything. It plays the noise, but nothing happens, and my mana is not consumed. Am I using it correctly? 

Video showing what happens: https://filebin.net/us1odzq9ldsp19yv
